reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -42,22 +45,32 @@ export default function WorkflowList({ workflows, refreshedAt }) {
           duration,
           completed,
           total,
+          size,
           status,
-        }) => (
+        } = workflow;
+        return (
           <div
             key={id}
             onClick={() => history.push(`/details/${id}`)}
             className={`${styles["workflow"]} ${
               status === "deleted" ? styles["deleted"] : ""
             }`}
           >
-            <div>
+            <div className={styles["details-box"]}>
               <Icon
                 name={statusMapping[status].icon}
                 color={statusMapping[status].color}
               />{" "}
-              <span className={styles["name"]}>{name}</span>
-              <span className={styles["run"]}>#{run}</span>
+              <span className={styles.name}>{name}</span>
+              <span className={styles.run}>#{run}</span>
+              <span
+                className={`${styles.size} ${
+                  status === "deleted" && size !== "0K" ? styles.highlight : ""
```

Comment:
we have many checks for `status === "deleted" ` while rendering the component. I would propose to create a new variable (something like `statusDeleted`) with this check in line 50 and use it while rendering

Reply:
we could even have another one for `status === "deleted" && size !== "0K"` just to give it a name and to be more declerative